app/views/admin/organization_members/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :header do %>
+  <%= breadcrumbs "Organizations", admin_organizations_path, @organization.name, admin_organization_path(@organization) %>
+  <%= index_header "New Organization Member" %>
+<% end %>
+
+<%= form_with(model: [:admin, @organization, @organization_member], builder: SpectreFormBuilder) do |form| %>
+  <%= form.errors %>
+
+  <%= form.text_field :full_name %>
+
+  <%= form.fields_for :users_attributes, @user do |user_form| %>
+    <%= user_form.text_field :email, type: "email" %>
+  <% end %>
+
+  <% role_options = OrganizationMember.roles.values.sort.map { |role| [role.capitalize, role] } %>
+  <%= form.select :role, options_for_select(role_options, @organization_member.role) %>
+
+  <div>
+    <%= form.submit %>
+  </div>
+<% end %>
```

Comment:
The `new` and `edit` templates could share a `form` partial, but there'd need to be a decent amount of logic in it (figuring out the form url, whether or not to include the email field) so this might be a small amount of duplication but I think it's a bit easier to reason about.

Reply:
I agree that the duplication is easier to work with than a lot of template logic.